### DISCUSSION TITLE
Voice analysis to display a warning if talking with microphone off

### DISCUSSION THREAD

**Discussion by CoBC** [BearWare/TeamTalk5#1474] [Ideas]
Some users are sometimes confused because they're talking but the microphone is off. If we enable a voice analysis by WebRTC, we can display a warning if we're talking with microphone off, but this feature will require a bit of CPU all time.
This discussion is here to request users point of view about this feature, what do you think?

**Comment by lbk2907** [comment]
Is good to have this feature. It will be grate if user have an option to
turn on this feature or not.

**Comment by Nardol** [comment]
Good idea, also if it is configurable as usual.
